Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1010-TPDM-ResourceClaims.sql

```diff
@@ -2331,6 +2331,45 @@ BEGIN
             END
         END
   
+    ----------------------------------------------------------------------------------------------------------------------------
```

Comment:
@stevenarnold-ed-fi  did this get removed by mistake in a earlier PR and is being added back? This resource was there in 0.8?